views/vm/create.erb

```diff
@@ -73,9 +73,13 @@
                   "components/form/checkbox",
                   locals: {
                     name: "enable-ip4",
-                    label: "Enable Public IPv4",
-                    value: "1",
-                    selected: "1"
+                    label: "Public IPv4 Support (+<span class='enable-ip4-1-monthly-price'>-</span><span class='text-xs text-gray-500'>/mo</span>)",
+                    options: [
+                     ["1", "Enable Public IPv4", "location-based-price", {"data-amount" => "1", "data-resource-type" => "IPAddress", "data-resource-family" => "IPv4"}]
+                    ],
+                    selected: "1",
+                    description:
+                      "Assigning a public IPv4 address to virtual machine allows to connect it via IPv4 or to allow virtual machine to connect public internet via IPv4."
```

Comment:
Should we mention that some resources you want to connect might not support communication from IPv6?

Reply:
It was my intention. We can make it more clear. Do you have any suggestion?

Reply:
No suggestion. I don't want to use a language like "You cannot..." also don't want to name the names :) but we need to communicate potential limitations. Maybe something like;

> Without an IPv4 address, some web sites might not work properly if they are not supporting IPv6.

One thing to consider is to use something other than "web sites" as the internet is not just consist of web sites, but I'm not sure what would be a better terminology.

Reply:
@furkansahin Do you have any comment for it?

Reply:
we can just say;
```
must have to access ipv4 only networks, such as, github.com.
```

Reply:
I updated as "Public IPv4 address is required to access IPv4 only networks, such as github.com."
<img width="917" alt="Screenshot 2023-08-09 at 16 13 56" src="https://github.com/ubicloud/ubicloud/assets/993199/f0490d59-cf60-4127-b796-1f8e01c937a4">

Reply:
Do we want to specifically mention github.com?

Reply:
Lots of people think GitHub is the biggest barrier to switch only-IPv6 machines. But not sure to mention or not

Reply:
It mixed all feedbacks. What about this one?
> Needed for inbound and outbound public IPv4 connections. Websites that do not support IPv6 will be inaccessible without an IPv4 address.